dash_enterprise_auth/__init__.py

```diff
@@ -11,7 +11,11 @@
 import functools as _ft
 import json as _json
 
-import dash_core_components as _dcc
+import dash
+if hasattr(dash, "dcc"):
+    _dcc = dash.dcc
```

Comment:
```suggestion
import dash as _dash
if hasattr(_dash, "dcc"):
    _dcc = _dash.dcc
```
Just for consistency, as this module seems to want to be careful what its exports look like.